src/agent/state.ts

```diff
@@ -29,9 +29,7 @@ import * as v8Types from '../types/v8-types';
 import * as apiTypes from '../types/api-types';
 import {DebugAgentConfig} from './config';
 
-// TODO: Determine if `ScopeType` should be named `scopeType`.
-// tslint:disable-next-line:variable-name
```

Comment:
I would prefer to keep it capitalized as ScopeType. It is a foreign type, let's keep its original name.